scripts/update_metadata.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+'''
+@author: Patrick Cleeve
+
+This is a script to update the MD_POS_COR for the stage-bare component while odemis is running.
+
+run as:
+python3 ./scripts/update_metadata.py 
+
+and follow the instructions in the terminal.
+'''
+
+
+from odemis import model
+
+def main():
+
+    print(f"Calibration metadata updater")
+
+    stage = model.getComponent(role="stage-bare")
+    print(f"-"*20)
+    print(f"Current stage-bare component: {stage}")    
+    print(f"Current calibration metadata: {stage.getMetadata()[model.MD_POS_COR]}")
+
+    # get x , y values from user
+    x = input(f"Enter new x calibration value: ")
+    y = input(f"Enter new y calibration value: ")
+    print("-"*20)
+
+    # check they are floats
+    try:
+        x = float(x)
```

Comment:
How bad could things go if the value is incorrect? I'm a little worried the user types the values in mm instead of m. Could the stage go suddenly somewhere really bad? Maybe at least check that the values are "reasonable". For instance, check they are within the axes ranges, using `stage_bare.axes["x"].range`

Reply:
added, it might be a good idea to add something similar before the actual move too, as these values might be edited from the yaml too?